crates/fj-app/src/main.rs

```diff
@@ -45,18 +45,28 @@ fn main() -> anyhow::Result<()> {
 
     let args = Args::parse();
     let config = Config::load()?;
-
+    // model name defined here.
+    let fetch_model_name =
+        (args.model.as_ref().or(config.default_model.as_ref()))
+            .unwrap()
```

Comment:
This breaks when no model is defined at all, so neither via a command-line argument, nor as a default model in the configuration. This is a case that is otherwise handled, see the [`if let` in line 62](https://github.com/hannobraun/Fornjot/blob/45abaf5feff821e311406789043133598221a1ed/crates/fj-app/src/main.rs#L62) and [its `else` in lines 96-98](https://github.com/hannobraun/Fornjot/blob/45abaf5feff821e311406789043133598221a1ed/crates/fj-app/src/main.rs#L96-L98).

This can be reproduced like this:
1. While in your branch, install `fj-app` by running this from the repository root: `cargo install --path crates/fj-app/`
2. Run this from any directory that has no Fornjot model, nor Fornjot configurat: `fj-app`

This results in the following error:
```
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', crates/fj-app/src/main.rs:51:14
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Your code here duplicates the logic from line 62, which is a warning sign. Whenever you duplicate code like that, there's a danger that the two versions are different and one doesn't work correctly (as is the case here). Even if both work correctly, sooner or later one of the versions will be changed, and whoever does that might forget to update the other one.

The solution is to re-use the code that is already there. Instead of creating a new variable (`fetch_model_name`) here, why not move the code that formats the new error message into the `if let` (line 63 and following)? Then it can use the `model` variable that is already there.